rclpy/src/rclpy/node.hpp

```diff
@@ -42,6 +42,20 @@ get_node_name(py::capsule pynode);
  */
 const char *
 get_node_namespace(py::capsule pynode);
+
+/// Get a list of parameters for the current node
+/**
+ * On failure, an exception is raised and NULL is returned if:
```

Comment:
@sloretz safe to delete?

Reply:
Deleted this text in 33c40a34f62ff30c6490a03a31222d40250e0fc6